Mathlib/Algebra/Ring/ULift.lean

```diff
@@ -11,91 +11,91 @@ import Mathlib.Algebra.Ring.Equiv
 
 This file defines instances for ring, semiring and related structures on `ULift` types.
 
-(Recall `ULift α` is just a "copy" of a type `α` in a higher universe.)
+(Recall `ULift R` is just a "copy" of a type `R` in a higher universe.)
 
 We also provide `ULift.ringEquiv : ULift R ≃+* R`.
 -/
 
 
 universe u v
 
-variable {α : Type u}
+variable {R : Type u}
 namespace ULift
 
-instance mulZeroClass [MulZeroClass α] : MulZeroClass (ULift α) :=
-  { zero := (0 : ULift α), mul := (· * ·), zero_mul := fun _ => (Equiv.ulift).injective (by simp),
+instance mulZeroClass [MulZeroClass R] : MulZeroClass (ULift R) :=
```

Comment:
This looks wrong. Actually, this instance belongs in `Algebra.GroupWithZero.ULift`